miden/src/examples/blake3.rs

```diff
@@ -0,0 +1,80 @@
+use super::Example;
+use miden::{Assembler, DefaultHost, MemAdviceProvider, Program, StackInputs};
+use stdlib::StdLibrary;
+use test_utils::group_slice_elements;
+
+// EXAMPLE BUILDER
+// ================================================================================================
+
+pub fn get_example(n: usize) -> Example<DefaultHost<MemAdviceProvider>> {
+    // generate the program and expected results
+    let program = generate_blake3_program(n);
+    let expected_result = compute_hash_sequence(n);
+    println!(
+        "Generated a program to compute {}-th iteration of BLAKE3 1-to-1 hash; expected result: {:?}",
+        n, expected_result
+    );
+
+    Example {
+        program,
+        stack_inputs: StackInputs::try_from_values([4294967295; 8]).unwrap(),
```

Comment:
If we are using `[4294967295; 8]` as the initial value to hash, I'd create a constant for this.